src/types.ts

```diff
@@ -125,7 +125,7 @@ export type AsyncStorageStatic = {
   multiGet: (
     keys: string[],
     callback?: MultiGetCallback
-  ) => Promise<readonly KeyValuePair[] | void>;
+  ) => Promise<readonly KeyValuePair[]>;
```

Comment:
you'll also need to update the type of [returned promise here](https://github.com/react-native-async-storage/async-storage/blob/master/src/AsyncStorage.native.ts#L272)

Reply:
Thank you I missed it 😄